pkg/koordlet/runtimehooks/reconciler/reconciler.go

```diff
@@ -385,6 +445,35 @@ func (c *reconciler) reconcilePodCgroup(stopCh <-chan struct{}) {
 					}
 				}
 			}
+
+			for _, r := range globalCgroupReconcilers.allPodsLevel {
+				currentPods := make([]protocol.HooksProtocol, 0)
+				fns := make(map[string]reconcileFunc4AllPods)
```

Comment:
@saintube , I add a map here to store condition => reconcileFn and add a test too, PTAL.